ChemoTracker/Login.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { StyleSheet, Text, TextInput, Button, View, Alert, Image } from 'react-native';
+
+export default class Login extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Image
+          source={require('./chemotracker.png')}
+          style={styles.logo}
+        />
+        <TextInput
+          style={styles.textFieldContainer}
+          placeholder="email"
+        />
+       <TextInput
+         style={styles.textFieldContainer}
+         placeholder="password"
+       />
+       <View style={styles.buttonContainer}>
+         <Button
+          onPress={()=> {}}
+          title="Login"
+          color="#fff"
+          accessibilityLabel="Tap on Me"
+         />
+       </View>
+       <Text>New? <Text style={{color: "#6374f4", fontWeight: 'bold'}}>SIGN UP</Text></Text>
```

Comment:
I think all styling should be in the `Stylesheet`, and let's put it in a separate file